### PR TITLE
Add Preact to the scaffolding app

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by helfi92** [comment]
I had to change the preset slightly to make it work. Changed [L15](https://github.com/fa7ad/neutrino-preset-preact/blob/master/index.js#L15) to `{ pragma: 'h' }`.

**Comment by eliperelman** [comment]
@helfi92 I think that's because of your import:

```js
import { h, Component } from 'preact';
```

I think if you instead do:

```js
import Preact, { Component } from 'preact';
```

Then the pragma is different. I'm actually not sure what the recommended pattern here is. @developit do you have advice here?

**Comment by helfi92** [comment]
In [boilerplate code](https://github.com/developit/preact-boilerplate/blob/master/src/index.js#L3) that @developit provides,  `import { h, Component } from 'preact';` is used. I assume it's the recommended one.

**Comment by eliperelman** [comment]
@helfi92 ah, good find.

**Comment by eliperelman** [comment]
Also useful:

https://github.com/developit/preact-cli/blob/master/src/lib/babel-config.js#L27-L31

File: packages/create-neutrino-project/commands/init/templates/neutrino-preset-preact/src/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { h, render } from 'preact';
+
+let root;
```
[the remark this answers is not in the document]
Comment (helfi92) [reply packages/create-neutrino-project/commands/init/templates/neutrino-preset-preact/src/index.js]: @eliperelman HMR fails to work using this approach.

[the remark this answers is not in the document]
Comment (helfi92) [reply packages/create-neutrino-project/commands/init/templates/neutrino-preset-preact/src/index.js]: Also, to be consistent with the other templates we offer, perhaps let's not have `document.getElementById('root')` stored in a variable.

[the remark this answers is not in the document]
Comment (eliperelman) [reply packages/create-neutrino-project/commands/init/templates/neutrino-preset-preact/src/index.js]: > HMR fails to work using this approach.

lol 😭 

> Also, to be consistent with the other templates we offer, perhaps let's not have document.getElementById('root') stored in a variable.

Works for me. It did just make me think of something: that we hardcode `root` even though it  can be overridden with `appMountId`. In the future, we could add this to the template.

[the remark this answers is not in the document]
Comment (helfi92) [reply packages/create-neutrino-project/commands/init/templates/neutrino-preset-preact/src/index.js]: Agreed :)

**Comment by eliperelman** [comment]
I think this is good for now. We can follow up with auto-importing of `h`.

**Pull request merged**